samples/interop/src/main/kotlin/com/slack/circuit/sample/interop/FlowCounterPresenter.kt

```diff
@@ -2,39 +2,76 @@
 // SPDX-License-Identifier: Apache-2.0
 package com.slack.circuit.sample.interop
 
+import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.collectAsState
 import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import app.cash.molecule.RecompositionClock
+import app.cash.molecule.launchMolecule
 import com.slack.circuit.runtime.presenter.Presenter
 import com.slack.circuit.runtime.presenter.presenterOf
 import com.slack.circuit.sample.counter.CounterScreen
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.channels.Channel
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.receiveAsFlow
+import kotlinx.coroutines.launch
+
+/** A basic [Flow]-based presenter interface. */
+fun interface FlowPresenter<T : Any, E : Any> {
```

Comment:
Minor suggestion: consider swapping `E` and `T` here. They feel odd and backwards given that `present` accepts a `Flow<E>` and returns a `SateFlow<T>`. Especially when reading a signature that returns `FlowPresenter` (such as `asFlowPresenter` below)

Reply:
let me make it just match what we use elsewhere

Reply:
[90233bf](https://github.com/slackhq/circuit/pull/555/commits/90233bf47b94faa0d01ac7c2e8465d826460ee7a)